### ISSUE TITLE
Update configuration.md with gRPC config

### ISSUE THREAD

**Issue by rshsmi** [prometheus/blackbox_exporter#893]
addition of gRPC config to configuration.md

### <grpc_probe>

```yml

# The IP protocol of the gRPC probe (ip4, ip6).
[ preferred_ip_protocol: <string> ]

# Configuration for TLS protocol of gRPC probe.
tls_config:
  [ <tls_config> ]
```

addition of gRCP support to Readme

HTTP, HTTPS (via the `http` prober), DNS, TCP socket, ICMP and gRPC (see permissions section) are currently supported.
Additional modules can be defined to meet your needs.

**Comment by roidelapluie** [comment]
Thanks! Are you willing to submit a pull request?

**Comment by rshsmi** [comment]
Sure, I'm working on it.

**Comment by roidelapluie** [comment]
Thanks, fixed in #894
